Makefile

```diff
@@ -31,11 +31,8 @@ GO15VENDOREXPERIMENT := 1
 PKGS := $(shell go list -tags "${DOCKER_BUILDTAGS}" ./... | grep -v ^github.com/docker/distribution/vendor/)
 
 # Resolving binary dependencies for specific targets
-GOLINT_BIN := $(GOPATH)/bin/golint
-GOLINT := $(shell [ -x $(GOLINT_BIN) ] && echo $(GOLINT_BIN) || echo '')
-
-GODEP_BIN := $(GOPATH)/bin/godep
-GODEP := $(shell [ -x $(GODEP_BIN) ] && echo $(GODEP_BIN) || echo '')
+GOLINT := $(shell which golint || echo '')
```

Comment:
@stevvooe: I left this in to preserve the
```
	$(if $(GOLINT), , \
 		$(error Please install golint: `go get -u github.com/golang/lint/golint`))		 		```

I'm not sure how to do that without calling `which` or doing something similar to search $PATH. If you have any suggestions, let me know.